sequential_parallel.py

```diff
@@ -0,0 +1,15 @@
+from threading import Thread
+
+
+def start_parallel(functions, arguments):
```

Comment:
We might want the option to add 'timeout' as a parameter to this function.
It can help us avoid threads that run forever.

Reply:
I add an optional argument for the function class. The user has control of how long each function runs.